lib/webpack-cli.js

```diff
@@ -32,9 +33,21 @@ class webpackCLI extends GroupHelper {
 
     checkDefaults(groupResult) {
         const { options } = groupResult;
-        if (!options.entry) {
-            this.shouldUseMem = true;
+        if(options.entry && options.entry === './index.js') {
```

Comment:
Sometimes people don't set the `.js`. What should we do in this case?

Reply:
We gotta check for `index` too, but we will treat it as a directory, so if the user supplies `index`, we will look for the index directory and a filename under that dir. If the filename isn't present, we will set `index/bundle.js`

Reply:
If it's a directory, it should be treated as `index/index.js`